app/task/project/loadotio/loadotio.cpp

```diff
@@ -88,11 +89,36 @@ bool LoadOTIOTask::Run()
 
   // Keep track of imported footage
   QMap<QString, Footage*> imported_footage;
+  QMap<OTIO::Timeline*, Sequence*> timeline_sequnce_map;
 
+  // Variables used for loading bar
+  float number_of_clips = 0;
+  float clips_done = 0;
+
+  // Generate a list of sequences with the same names as the timelines.
+  // Assumes each timeline has a unique name.
   foreach (auto timeline, timelines) {
-    // Create sequence
     Sequence* sequence = new Sequence();
     sequence->SetLabel(QString::fromStdString(timeline->name()));
```

Comment:
Might be good to check if `timeline->name()` is empty, and if so, use an auto-generated name like "Sequence 1", "Sequence 2", etc. Otherwise it potentially permits a sequence with an empty name which Olive code assumes will never happen.